packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -991,10 +992,13 @@ export const prepareVaultManagerKit = (
             AmountMath.add(state.totalCollateral, vault.getCollateralAmount()),
             oldCollateral,
           );
+
+          // add in new debt; subtract old debt
```

Comment:
I find this comment confusing. not a blocker.
```suggestion
          // // total += vault's delta (post — pre)
```
or remove the newline so the comment on `totalCollateral` applies